appeals/api/src/server/endpoints/lpa-questionnaires/lpa-questionnaires.validators.js

```diff
@@ -67,7 +69,23 @@ const patchLPAQuestionnaireValidator = composeMiddleware(
 
 			return value;
 		}),
-	validateDateParameter('lpaQuestionnaireDueDate'),
+	validateDateParameter(
+		'lpaQuestionnaireDueDate',
+		(
+			/** @type {any} */ value,
+			/** @type {{ req: { body: { validationOutcome: string } } }} */ { req }
+		) => {
+			if (value && !isOutcomeIncomplete(req.body.validationOutcome)) {
+				throw new Error(ERROR_ONLY_FOR_INCOMPLETE_VALIDATION_OUTCOME);
+			}
```

Comment:
We should probably have a test to check that the correct error message is returned if `lpaQuestionnaireDueDate` is in the request body when the outcome is `Valid`

Reply:
Done